docs/includes/start-web-sideload-instructions.md

```diff
@@ -0,0 +1,12 @@
+```command&nbsp;line
+npm run start:web -- --document {url}
+
+// examples:
+npm run start:web -- --document https://contoso.sharepoint.com/:t:/g/EZGxP7ksiE5DuxvY638G798BpuhwluxCMfF1WZQj3VYhYQ?e=F4QM1R
+
+npm run start:web -- --document https://1drv.ms/x/s!jkcH7spkM4EGgcZUgqthk4IK3NOypVw?e=Z6G1qp
+
+npm run start:web -- --document https://contoso-my.sharepoint-df.com/:t:/p/user/EQda453DNTpFnl1bFPhOVR0BwlrzetbXvnaRYii2lDr_oQ?e=RSccmNP
+```
```

Comment:
I wonder if this would look better with the code broken up a little bit. Right now, the copy button gets all the examples:

![image](https://user-images.githubusercontent.com/38896772/149597705-3749e70a-742c-49f5-9105-7836a84bc42d.png)


What if instead the top line was copyable, and the others were inline code?

```command&nbsp;line
npm run start:web -- --document {url}
```
Some URL examples include:
- `npm run start:web -- --document https://contoso.sharepoint.com/:t:/g/EZGxP7ksiE5DuxvY638G798BpuhwluxCMfF1WZQj3VYhYQ?e=F4QM1R`
- `npm run start:web -- --document https://1drv.ms/x/s!jkcH7spkM4EGgcZUgqthk4IK3NOypVw?e=Z6G1qp`
- `npm run start:web -- --document https://contoso-my.sharepoint-df.com/:t:/p/user/EQda453DNTpFnl1bFPhOVR0BwlrzetbXvnaRYii2lDr_oQ?e=RSccmNP`

(The GitHub suggestion feature does not like nested code blocks...)

Reply:
I like this idea. I've made the change.